src/backend/distributed/transaction/commit_protocol.c

```diff
@@ -197,6 +260,8 @@ CommitRemoteTransactions(List *connectionList, bool stopOnFailure)
 											  command->data, nodeName, nodePort)));
 				}
 			}
+
+			ereport(DEBUG2, (errmsg("sent COMMIT PREPARED over connection %ld", connectionId)));
```

Comment:
If we want to show something here, it might be nice to just show the command that was sent, such that the user can see the transactions that were prepared/committed.